templates/splunk-enterprise.template

```diff
@@ -498,6 +605,35 @@
                 ]
             }
         },
+        "SplunkHttpEventCollectorLoadBalancerSecurityGroup": {
+            "Type": "AWS::EC2::SecurityGroup",
+            "Properties": {
+                "VpcId": {
+                    "Ref": "VPCID"
+                },
+                "GroupDescription": "Enable port 8088 on ELB for HEC input",
+                "SecurityGroupIngress": [
+                    {
+                        "IpProtocol": "tcp",
+                        "FromPort": "8088",
+                        "ToPort": "8088",
+                        "CidrIp": "0.0.0.0/0"
```

Comment:
Do we really need to open this port to world?

Reply:
HTTP Event Collector endpoint requires a token to accept data, so it's not completely exposed. I agree with you however that it's better not to make public access the default. I'll turn this CIDR block into an additional user parameter.